src/handlers/payout/index.ts

```diff
@@ -44,7 +48,7 @@ export const handleIssueClosed = async () => {
   if (!recipient) {
     if (issue.state_reason === StateReason.COMPLETED) {
       logger.info(`Recipient address is missing`);
-      await addCommentToIssue(`@${assignee.login} would you please post your wallet address here?`, issue.number);
+      await addCommentToIssue(`Please update your wallet address using \`wallet 0x..4DF\`\nand re-request permit generation\n@${assignee.login}`, issue.number);
```

Comment:
I am just curious whether we have a mechanism for a hunter to re-request permit generation lol

Reply:
Re-request here means asking the members to complete the permit generation,
But we might have a command, that'd be great.

Reply:
> I am just curious whether we have a mechanism for a hunter to re-request permit generation lol

This will be complex to implement because if a permit is used (bounty hunter withdraws 100 DAI) and they keep regenerating permits, then they can keep taking out 100 DAI. 

Unless we use the issue number as a nonce or something?

Reply:
@rndquu did the most research on permit generation. I'll let them handle creating the issue if there's a plan/proposal for the solution.